### DISCUSSION TITLE
Same set of badges across different dashboards

### DISCUSSION THREAD

**Discussion by bpottle1** [home-assistant/frontend#21710] [Dashboard]
The new badges are great! I think it would be super helpful though to have the option to have a set of badges be duplicated and kept in sync between different pages in a dashboard. For instance, I typically use badges to show how many lights, fans, etc. are on in the house. I would like that set of badges to be duplicated across each page in my dashboard so I don't have to manually create the same badges over and over. 

This would also help with maintenance and updating my badges later on. If I decided to change or add badges, I have to go to each different page to do so. Having the option for a page to mirror the badges from another page would be great!

**Comment by fkolacek** [comment]
I would also like the ability to share badge definitions across multiple dashboards or views. This would greatly reduce the complexity of my dashboards and eliminate duplicated code, making maintenance much easier.

Ideally this would be possible to configure in the UI, but it could be also "as easy" as being able to do:

`badges: !include "badges.yaml"`

There is a similar (just now realized very old) post[1] about being able to do similar things with cards as well (e.g. using !include with cards as well).

[1] https://community.home-assistant.io/t/cards-on-multiple-views-and-dashboards/236717/10
